### ISSUE TITLE
Can't select first and second tabs in "Tabbed View"

### ISSUE THREAD

**Issue by psavery** [HEXRD/hexrdgui#261]
In the raw image view, if "Tabbed View" is checked, the user can switch between the third and fourth tabs if there are four images. But they can't switch to the first and second tabs (clicking the tabs seems to not do anything).

I'm not sure if anyone uses the tabbed view, but I thought I'd at least record this issue.

**Comment by psavery** [comment]
Update: this actually only appears to be a problem in #260.

**Comment by psavery** [comment]
The issue seems to be caused by the images being loaded and drawn before the Qt event loop has started. It seems like a strange issue, and I'm not sure what the problem exactly is.

**Comment by psavery** [comment]
So, this bug is not really an issue anymore, because some changes in #260, that were needed anyways, fix the issue.

However, I dug just a little deeper here to get more information in case we run into this bug in the future. I found out that the bug occurs if [this line](https://github.com/HEXRD/hexrdgui/blob/8d214cf3bf5bade4c72a37fee432be9877157e45/hexrd/ui/image_tab_widget.py#L119-L120) is called before the Qt event loop has started. That `CustomNavigationToolbar2QT` class inherits from matplotlib's `NavigationToolbar2QT` class, and I confirmed that the bug still occurs if we use matplotlib's `NavigationToolbar2QT` class instead of our custom one. So, it appears that this bug might be caused by some kind of bad interaction between matplotlib and pyside2, before the Qt event loop has started.

Anyways, because the issue is not occurring for us anymore due to updates in #260, I am probably not going to pursue this further right now. But I thought I'd at least record my findings in case we run into this in the future.

The bug is occurring on python 3.7, for the latest matplotlib (matplotlib-3.1.3), and the latest PySide 2 (5.14.1).
